app/user/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './userprofile/userprofile.component';
```

Comment:
We don't use `index.ts` files anymore - they aren't recommend by the latest Angular 2 CG.